modules/core/src/main/scala/gem/math/Angle.scala

```diff
@@ -144,6 +159,14 @@ object Angle {
   def fromDoubleRadians(rad: Double): Angle =
     fromDoubleDegrees(rad.toDegrees)
 
+  /** Attempt to parse an [[Angle]] from a `.formatDMS`-formatted string. */
+  def unformatDMS(s: String): Option[Angle] =
```

Comment:
Yeah `parse` is probably better. I wanted them to be parallel but it's kind of irritating. I'll rename them,